components/badges/badges.css

```diff
@@ -0,0 +1,65 @@
+.badge-container {
+  font-size: 2.5rem;
+  width: fit-content;
+  height: fit-content;
+  position: relative;
+  display: inline-block;
+}
+.badge-number {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 2rem;
+  min-width: 2rem;
+  border-radius: 50%;
+  font-size: 1.2rem;
+  color: var(--secondary-font-color);
+  background-color: var(--primary-color);
+  position: absolute;
+}
+.badge-top-right {
+  right: -1.2rem;
+  bottom: 2rem;
+}
+.badge-top-left {
+  top: -5px;
+  left: -1rem;
+}
```

Comment:
I would recommend that if you are specifying values in rems you should not use px in top: -5px

Reply:
During the code reviews with MA's it was suggested to use px where value is less than 1 rem, that's why I have used px here.